controller/controller.go

```diff
@@ -1235,8 +1235,10 @@ func (ctrl *ProvisionController) shouldProvision(ctx context.Context, claim *v1.
 // shouldDelete returns whether a volume should have its backing volume
 // deleted, i.e. whether a Delete is "desired"
 func (ctrl *ProvisionController) shouldDelete(ctx context.Context, volume *v1.PersistentVolume) bool {
+	klog.V(4).Infof("shouldDelete Volume : %+v", volume)
 	if deletionGuard, ok := ctrl.provisioner.(DeletionGuard); ok {
```

Comment:
I don't think the csi-provisioner implements DeletionGuard. Maybe the log should be one level higher?

Reply:
If csi provisioner doesn't use deletion guard, I will remove that log statement